spec/views/orders/edit.html.erb_spec.rb

```diff
@@ -2,6 +2,23 @@
 
 RSpec.describe "orders/edit", type: :view do
   before(:each) do
+    @drivers = assign(:drivers, [
+      Driver.create!(
+      :name => 'Ivan',
+      :phone_number_string => '09365479815',
+      :status => 0,
+      :email => 'Ivan@mail.ru',
```

Comment:
Please, use `example.com` emails for tests.
Also I recommend you to use [factories](https://github.com/thoughtbot/factory_girl_rails) with [auto-generated faked values](https://github.com/stympy/faker), not the hard-coded examples.